test/test_torch.py

```diff
@@ -1036,8 +1036,8 @@ def compare(t, k, dim, dir):
                        random.randint(1, SIZE),
                        random.randint(1, SIZE))
 
-        for kTries in range(3):
-            for dimTries in range(3):
```

Comment:
The linter I have recommends using _ before unused variables in loops. But obviously this can be changed.